openmdao/docs/developer_docs/repo_guide/github.rst

```diff
@@ -0,0 +1,65 @@
+.. _`github_pages`:
+
+github-pages Repository Setup
+=============================
+
+If you want to publish to github-pages, from your docs built on Travis CI, make sure your Github repository is set up to be public,
+otherwise some of the instructions in this document may not work for you.
+
+Personal Access Token
+---------------------
+
+A Personal Access Token is something created on Github, to allow another service (e.g. Travis) to access Github,
+with a certain amount of permission.  Once you create a Personal Access Token, it will be visible for
+one viewing, and you must copy it then use it immediately before you lose it (and then you have to make another one).
+
+To create a Personal Access Token:
+Log in to githb.com.  Go to Github *account* Settings (**not** Repo Settings)->Dev Settings->Personal Access Token
+The menu will look like this:
+
+.. image::
+    images/AccessToken1.png
+
+Give the access token the name GH_TOKEN, and the permission "repo access," as pictured:
+
+.. image::
+    images/AccessToken2.png
+
+You'll need this token to allow the transfer of your documents to github-pages, once they are built on Travis CI.
+
+Then on Travis CI, you'll need to go to your Environment Variables section, and create a new env var, as shown here:
```

Comment:
suggest: "env var" -> "variable" (since it's clear from context that you're talking about environment variables.)